lib/nitra/command_line.rb

```diff
@@ -14,6 +14,22 @@ def initialize(configuration, argv)
           configuration.burndown_report = filename
         end
 
+        opts.on("--rspec-formatter FORMATTER", String, "Additional rspec formatter to use.") do |formatter_name|
+          configuration.rspec_formatter = formatter_name
+        end
```

Comment:
I understand that specs in this project are a bit out of whack. If you have time to support the new formatter and output options with specs that would be 💯 I'll leave it to your discretion tho.